samples/browser/src/main/java/org/mozilla/samples/browser/BrowserFragment.kt

```diff
@@ -104,6 +107,18 @@ class BrowserFragment : Fragment(), BackHandler {
         promptFeature = PromptFeature(components.sessionManager, requireFragmentManager())
 
         windowFeature = WindowFeature(components.engine, components.sessionManager)
+
+        featureManager.apply {
+            register(sessionFeature)
```

Comment:
I guess `register` could take a vararg? Then we could call `register(sessionFeature, toolbarFeature)` etc.

Reply:
There could also be an `unregisterAll`